dandi/cli/cmd_organize.py

```diff
@@ -140,7 +163,7 @@ def act(func, *args, **kwargs):
                 "Only 'dry' or 'move' mode could be used to operate in-place "
                 "within a dandiset (no paths were provided)"
             )
-        lgr.info(f"We will organize {dandiset_path} in-place")
+        lgr.info("We will organize %s in-place", dandiset_path)
```

Comment:
I am ok either way here but note that it is generally preferable to use original `%` + args notation since then string is not even interpolated unless logging level is "enabled".  With `f""` strings you would always cause string interpolation (and possibly relatively expensive `str`) even if end message would not even be displayed

Reply:
More importantly, if you use f-strings and one of the interpolated strings contains a `%`, the logger will still try to format it, resulting in an error.